src/springfield/fields.py

```diff
@@ -390,12 +390,26 @@ def __init__(self, entity, *args, **kwargs):
                        Use 'self' to use the :class:`Entity` class that
                        this field is already bound to.
         """
-        self.type = entity
+        self._type = entity
         super(EntityField, self).__init__(*args, **kwargs)
 
+    @property
+    def type(self):
+        if not callable(self._type):
```

Comment:
The only type that will work for import is a sting type. Something like `1` is not a callable, but not importable. So for this case, checking for string type is best.